src/file-size-using-promises.js

```diff
@@ -0,0 +1,48 @@
+import fs from "fs"
+
+export function sizeOfFileAtPathUsingPromises(path) {
+  return new Promise((resolve, reject) => {
+    fs.stat(path, (err, stats) => {
```

Comment:
Nice try!, now instead of using callbacks directly, can you first promisify them first and see how the implementation becomes more straightforward?

Reply:
Done

Reply:
Done